bases/recommended.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Recommended",
+  
+  "compilerOptions": {
+    /** Strict mode offers the most safety for your application */
+    "strict": true,
+    
+    /** These three options are to improve compatability with the js + babel ecosystem */
+    "isolatedModules": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    
+    /** Improves `tsc` launch speed in favour of some accuracy in your @types **/
+    "skipLibCheck": true,
```

Comment:
The docs for `skipLibCheck` seem to be different than what your comment says here.

The docs say that this property will "Skip type checking of _all_ declaration files (*.d.ts)."

https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
Yeah I can confirm this option turns off type checking of hand-written DTS files in your project, unfortunately. As a result I find it difficult to recommend.

Reply:
Yeah, it's a good point - I was thinking of `types` at the same time. We opted to recommend it for speed and that most dts files come from `@types` where we have tonnes of testing infra (and that most folks won't go fix these things upstream) - we'd probably fix the issues, others would get stuck

Reply:
That sounds like the common case - where the declarations files are for dependencies as opposed to defining types for files inside your project.

I know this isn't the right place to suggest it, but maybe the implementation of `skipLibCheck` could be updated to make this distinction.

Reply:
Not a bad call IMO